client/src/modules/Job/theme/colors.ts

```diff
@@ -0,0 +1,6 @@
+export const COLORS = {
+  Neutral_9: '#434343',
+  RS_yellow: '#FFDB20',
+  white: '#FFFFFF',
+  Neutral_2: '#FAFAFA',
```

Comment:
1) I would suggest to use enum `here`
2) Please consider to make follow consistent codestyle for object props/enum members names

Reply:
fixed it